apps/condo/domains/user/components/auth/InputPhoneForm.tsx

```diff
@@ -50,6 +52,9 @@ export const InputPhoneForm: React.FC<InputPhoneFormProps> = ({ onFinish }) => {
     const { query: { next } } = router
     const redirectUrl = (next && !Array.isArray(next) && isSafeUrl(next)) ? next : '/'
 
+    const { organization } = useOrganization()
+    const country = get(organization, 'country', defaultLocale)
```

Comment:
It is better to not use lodash get on the client – it messes up typescript types 

```suggestion
    const country = organization.country || defaultLocale
```